src/wallet/rpcdump.cpp

```diff
@@ -474,15 +474,15 @@ UniValue dumpwallet_impl(const UniValue& params, bool fHelp, bool fDumpZKeys)
     } catch (const std::runtime_error& e) {
         throw JSONRPCError(RPC_INTERNAL_ERROR, e.what());
     }
-    if (exportdir.empty()) {
+    /* if (exportdir.empty()) {
```

Comment:
Please don't comment out code, please just remove it.

Reply:
Removed